ndnfs/daemon/driver.go

```diff
@@ -80,7 +245,22 @@ func (d NdnfsDriver) Mount(r volume.MountRequest) volume.Response {
 	log.Info(DN, "Mount volume: ", r.Name)
 	d.Mutex.Lock()
 	defer d.Mutex.Unlock()
-	mnt, err := d.Client.MountVolume(r.Name)
+	var err error
+	var mnt string
+
+	nfs := fmt.Sprintf("%s:/%s", d.Config.Nedgedata, r.Name)
+	mnt = filepath.Join(d.Config.Mountpoint, r.Name)
+	log.Debug(DN, "Creating mountpoint folder: ", mnt)
+	if out, err := exec.Command("mkdir", "-p", mnt).CombinedOutput(); err != nil {
```

Comment:
I do not think it should throw anything. But if you remove 'if err ==' at the end, it may say that the variable declared at 'var err errror' is not used.